third_party/islet/include/islet.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved.
```

Comment:
Oh, I see this copyright here. Do you [not] need to mention the license under which this is copyrighted?
Similar copyright notice may be needed in `.cc`, `.c` file(s).